### PR TITLE
feat(console): delete connector

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add feature: delete connector. Though the label is "delete", but actually it will be "disabled".

## Linear Issue Reference

LOG-1869

## Testing

截屏2022-03-22 下午1 59 55

### CODE REVIEW

**Review by simeng-li — APPROVED** [review]
LGTM

File: packages/console/src/pages/ConnectorDetails/index.tsx [hunk 1]
```diff
@@ -74,6 +75,24 @@ const ConnectorDetails = () => {
...
+    if (!connectorId) {
+      return;
+    }
+
+    await api
+      .patch(`/api/connectors/${connectorId}/enabled`, {
+        json: { enabled: false },
+      })
+      .json<ConnectorDTO>();
+
```
Comment (xiaoyijun) [inline packages/console/src/pages/ConnectorDetails/index.tsx:88]: Need a toast on delete succeeded.

Comment (xiaoyijun) [inline packages/console/src/pages/ConnectorDetails/index.tsx:88]: Need a toast on delete succeeded.

**Review by xiaoyijun — CHANGES_REQUESTED** [review]

File: packages/console/src/pages/ConnectorDetails/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/ConnectorDetails/index.tsx:89]: updated

**Review by xiaoyijun — APPROVED** [review]

**Pull request merged**